autoinstall-schema.json

```diff
@@ -281,6 +281,9 @@
                 },
                 "sources": {
                     "type": "object"
+                },
+                "disable_components": {
+                    "type": "array"
```

Comment:
specifically it's an array of strings? Does it make sense to only accept restricted/main/universe/multiverse here?

Reply:
good idea, fixed